src/CreatorToken.sol

```diff
@@ -20,61 +33,115 @@ contract CreatorToken is ERC721Royalty {
   error CreatorToken__MinHoldingTimeNotReached(uint256 holdingTime, uint256 minHoldingTime);
   error CreatorToken__ContractIsPaused();
 
+  /// @notice ID of the last token minted.
   uint256 public lastId;
+  /// @notice Total supply of the creator tokens.
   uint256 public totalSupply;
+  /// @notice Address of the creator of the creator token.
   address public creator;
+  /// @notice Address of the admin of the creator token.
   address public admin;
+  /// @notice Address of the referrer, if any.
   address public immutable REFERRER;
+  /// @notice State indicating whether the contract is paused.
   bool public isPaused;
+  /// @notice URI pointing to the metadata for the creator token.
   string private creatorTokenURI;
+  /// @notice ERC20 token used for payments in the contract.
   IERC20 public payToken;
+  /// @notice Bonding curve contract used to determine token prices.
   IBondingCurve public immutable BONDING_CURVE;
 
+  /// @notice Constant to represent percentages in basis points.
   uint256 constant BIP = 10_000;
+  /// @notice Creator fee taken as a percentage when tokens are bought, in basis points.
   uint256 public immutable CREATOR_FEE_BIPS;
+  /// @notice Royalty fee for the creator, in basis points.
   uint96 public immutable CREATOR_ROYALTY_BIPS;
+  /// @notice Admin fee taken for the admin as a percentage, in basis points.
   uint256 public immutable ADMIN_FEE_BIPS;
+  /// @notice Maximum allowed fee in basis points.
   uint256 private constant MAX_FEE = 2500; // 25% in bips
+  /// @notice Minimum time a user must hold a token before selling, in blocks.
   uint256 private constant MIN_HOLDING_TIME = 60;
+  /// @notice Mapping to track purchase time of tokens.
   mapping(uint256 tokenId => uint256 blockTimestamp) internal purchaseTime;
 
-  event Bought(
+  /// @notice Event emitted when a new token is bought.
+  /// @param payer Address that paid for the token.
+  /// @param receiver Address that received the token.
+  /// @param tokenId ID of the token.
+  /// @param paymentAmount Amount of USDC paid for the token excluding fees.
+  /// @param creatorFee Amount of USDC paid to the creator.
+  /// @param adminFee Amount of USDC paid to the admin.
+  event Bought( // the address that paid for the token
```

Comment:
Looks like a leftover comment ended up on this line some how?

Reply:
yes! missed that. Fixed in [e159a53](https://github.com/showtime-xyz/creator-tokens/pull/46/commits/e159a53031291a7ef77626ca2d681aec08053372)